postcss.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  plugins: {
+    tailwindcss: {},
+    autoprefixer: {},
```

Comment:
Do we need autoprefixer? Is this a requirement of Tailwind or we adding it in as an extra dep?